### PR TITLE
Create WatchFile stream and simple server to server container logs ho…

### PR DESCRIPTION
**Pull request by nicktate**
@normanjoyner

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+            break;
+        case 'stderr':
+            responseStream = stderrStream;
+            break;
+        default:
+            res.status(400);
+            return next();
+    }
+
+    if (null === responseStream) {
```
Comment (normanjoyner) [inline app.js]: would this ever be triggered?

File: app.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+}
+
+class WatchStream extends Readable {
+    constructor(options, path) {
+        if (!path) {
+            path = options;
+            options = null;
+        }
+
+        options = options || {}
```
Comment (normanjoyner) [inline app.js]: semicolon

File: app.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+const _ = require('lodash');
+const express = require('express');
+const fs = require('fs');
+const helmet = require('helmet');
+const mkdirp = require('mkdirp');
+const stream = require('stream');
+const Readable = stream.Readable;
+
+const log_path = process.env.CS_LOG_PATH;
```
Comment (normanjoyner) [inline app.js]: we can grab this from existing `CS_PROC_OPTS` environment variable injected into our containers at runtime. `CS_PROC_OPTS` is a stringified json object including a key `container-log-dir`. Default value is: `/var/log/containership/applications`.

File: app.js (hunk 1 above)
Comment (nicktate) [inline app.js]: Yeah, if for some reason there is a stdout file but no stderr and they request stderr as the type. (probably not but its just a safety check)

**Pull request merged**